src/components/Projects/chatGPT-Projects.html

```diff
@@ -1,59 +1,47 @@
 <!DOCTYPE html lang="en">
 
 <head>
-  <!--- basic page needs
+
+    <!--- basic page needs
     ================================================== -->
-  <meta charset="utf-8" />
+    <meta charset="utf-8">
 
-  <!-- meta tags | https://ahrefs.com/blog/open-graph-meta-tags -->
-  <!-- mobile specific metas
+    <!-- meta tags | https://ahrefs.com/blog/open-graph-meta-tags -->
+    <!-- mobile specific metas
     ================================================== -->
-  <meta
-    name="viewport"
-    content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no"
-  />
+    <meta name='viewport' content='width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no'>
 
-  <meta
-    name="description"
-    property="og:description"
-    content="Code For Dallas public site"
-  />
+    <meta 
+        name="description" 
+        property="og:description" 
+        content="Code For Dallas public site"
+    />
 
-  <title>Code For Dallas</title>
-  <meta property="og:title" content="Code For Dallas Home Page" />
-  <meta name="author" content="Code For Dallas Volunteers" />
+    <title>Code For Dallas</title>
+    <meta property="og:title" content="Code For Dallas Hoem Page">
```

Comment:
This is made wrong again, so what changed? The old stuff in red was merged from your most recent PR.